test/test-step-06.bash

```diff
@@ -19,4 +19,4 @@ mv "$keep_file.keep" "$keep_file"
 
 # kcov causes this step to hang so skip the KCOV_DIR (probably the pm2 mathml2svg background process)
 SKIP_DOCKER_BUILD=1 \
-../enki --keep-data --data-dir $BOOK_DIR --command all-git-gdoc --start-at git-convert-docx
```

Comment:
This was a separate test to isolate the pm2 portion of docx builds. I remember having pm2 intermittently hang during the tests which was why it was done separately without KCOV_DIR set. **_Maybe_** something has changed since then that has fixed this issue; however, the way these tests are written now, steps 05 and 06 do exactly the same thing and one of them can be removed.

Reply:
ok thanks! Yeah, `pm2 kill` is [now](https://github.com/openstax/enki/blob/c77fb90cbdbd4b32aea354d87270b3af3e733265/dockerfiles/steps/git-convert-docx.bash#L35) being called which stops the step from hanging